agenda-webapps/src/main/webapp/vue-app/agenda/components/AgendaSchedule.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div>
+    <v-calendar
+      ref="calendar"
+      v-model="focus"
+      color="primary"
+      :events="eventList"
+      :event-color="getEventColor"
+      :type="type"
+      @click:event="showEvent"
+      @click:more="viewDay"
+      @click:date="viewDay" />
+    <agenda-event />
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    eventList: {
```

Comment:
`events` could be better than `eventList`

Reply:
done